api/src/components/publicationVersion/service.ts

```diff
@@ -352,11 +352,50 @@ export const updateStatus = async (id: string, status: I.PublicationStatusEnum)
         }
     });
 
+    const publication = await client.prisma.publication.findUnique({
+        where: { id: updatedVersion.versionOf },
+        select: { type: true }
+    });
+
     if (status === 'LIVE') {
-        // update "draft" links
+        if (publication?.type === 'PEER_REVIEW') {
+            // Check if any links created from this version have had the TO publication get a new live version since link creation.
+            // If so, alter the link to point to this new latest live version.
+            const outdatedDraftLinks = await client.prisma.links.findMany({
+                where: {
+                    publicationFromId: updatedVersion.versionOf,
+                    draft: true,
+                    versionTo: {
+                        isLatestLiveVersion: false
+                    }
+                }
+            });
+
+            for (const outdatedDraftLink of outdatedDraftLinks) {
```

Comment:
Potential to do this in a `promise.all` instead to save some time?

Reply:
In general yes, but since peer reviews can currently only link to one thing, I don't think it's necessary